### PR TITLE
test: clean/migrate some docker ps cli-only integration tests 

### PR DESCRIPTION
**Pull request by vdemeester**
Migrate some `docker ps` integration cli tests to unit tests or clean then as they are covered by unit/e2e tests on the `cli` side (see https://github.com/docker/cli/pull/880 and e2e tests on https://github.com/docker/cli)

🐯

### CODE REVIEW

**Review by dnephin — APPROVED** [review]
LGTM

File: integration-cli/docker_cli_ps_test.go [hunk 1]
```diff
@@ -815,23 +707,6 @@ func (s *DockerSuite) TestPsShowMounts(c *check.C) {
 	c.Assert(strings.TrimSpace(string(out)), checker.HasLen, 0)
 }
 
-func (s *DockerSuite) TestPsFormatSize(c *check.C) {
-	testRequires(c, DaemonIsLinux)
-	runSleepingContainer(c)
-
-	out, _ := dockerCmd(c, "ps", "--format", "table {{.Size}}")
```
Comment (thaJeztah) [inline integration-cli/docker_cli_ps_test.go]: This test may actually be testing that adding `{{.Size}}` to the format should automatically set the `--size` option; do we have a test covering that in the CLI?

File: integration-cli/docker_cli_ps_test.go [hunk 2]
```diff
@@ -512,48 +509,6 @@ func (s *DockerSuite) TestPsRightTagName(c *check.C) {
...
-	existingContainers := ExistingContainerIDs(c)
-	runSleepingContainer(c, "--name=first")
-	runSleepingContainer(c, "--name=second", "--link=first:first")
-
-	out, _ := dockerCmd(c, "ps", "--no-trunc")
-	lines := strings.Split(strings.TrimSpace(string(out)), "\n")
-	// strip header
-	lines = lines[1:]
-	lines = RemoveLinesForExistingElements(lines, existingContainers)
-	expected := []string{"second", "first,second/first"}
```
Comment (thaJeztah) [inline integration-cli/docker_cli_ps_test.go]: the `second/first` name is generated daemon-side; do we have an test for this?

**Review by thaJeztah — CHANGES_REQUESTED** [review]
left two questions 😅

File: integration-cli/docker_cli_ps_test.go (hunk 1 above)
Comment (vdemeester) [reply integration-cli/docker_cli_ps_test.go]: We should make sure we have that test yes, I'll update the test in the docker/cli 😉

File: integration-cli/docker_cli_ps_test.go (hunk 2 above)
Comment (vdemeester) [reply integration-cli/docker_cli_ps_test.go]: Nope, we need one 👼

File: integration-cli/docker_cli_ps_test.go (hunk 1 above)
Comment (thaJeztah) [reply integration-cli/docker_cli_ps_test.go]: Added in https://github.com/docker/cli/pull/880/files#diff-8b49518538a0aca318c2d28149b90dccR123

File: integration-cli/docker_cli_ps_test.go (hunk 2 above)
Comment (thaJeztah) [reply integration-cli/docker_cli_ps_test.go]: Added in https://github.com/moby/moby/pull/36330/files#diff-30649672f976017a15d8eb251ff4c751R49

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Comment by thaJeztah** [comment]
Janky failure doesn't look related; restarting

```
11:29:33 ----------------------------------------------------------------------
11:29:33 FAIL: check_test.go:366: DockerSwarmSuite.TearDownTest
11:29:33 
11:29:33 unmount of /tmp/docker-execroot/df153437b85ec/netns failed: invalid argument
11:29:33 unmount of /tmp/docker-execroot/df153437b85ec/netns failed: no such file or directory
11:29:33 check_test.go:371:
11:29:33     d.Stop(c)
11:29:33 daemon/daemon.go:391:
11:29:33     t.Fatalf("Error while stopping the daemon %s : %v", d.id, err)
11:29:33 ... Error: Error while stopping the daemon dfa7051d9d44f : exit status 130
11:29:33 
11:29:33 
11:29:33 ----------------------------------------------------------------------
11:29:33 PANIC: docker_api_swarm_test.go:261: DockerSwarmSuite.TestAPISwarmLeaderProxy
11:29:33 
11:29:33 [df153437b85ec] waiting for daemon to start
11:29:33 [df153437b85ec] daemon started
11:29:33 
11:29:33 [dfa7051d9d44f] waiting for daemon to start
11:29:33 [dfa7051d9d44f] daemon started
11:29:33 
11:29:33 [da895d950f5ec] waiting for daemon to start
11:29:33 [da895d950f5ec] daemon started
11:29:33 
11:29:33 [df153437b85ec] exiting daemon
11:29:33 [dfa7051d9d44f] daemon started
11:29:33 Attempt #2: daemon is still running with pid 4941
11:29:33 Attempt #3: daemon is still running with pid 4941
11:29:33 Attempt #4: daemon is still running with pid 4941
11:29:33 [dfa7051d9d44f] exiting daemon
11:29:33 ... Panic: Fixture has panicked (see related PANIC)
11:29:33 
```

**Pull request merged**